text/latex_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package text_test // import "gonum.org/v1/plot/text"
```

Comment:
We only have import comments in doc.go files (given we assume modules, we could actually get rid of them all together).